src/scripts/content/slack.js

```diff
@@ -1,12 +1,10 @@
 'use strict';
 /* global togglbutton, $ */
 
-togglbutton.render('#channel_name:not(.toggl)', { observe: true }, () => {
-  const placeholder = $('.channel_title_info');
-  const projectName = $('#team_name').textContent.trim();
-  const description = $('#channel_name')
-    .textContent.trim()
-    .replace(/^#/, '');
+togglbutton.render('[data-qa="channel_name"]:not(.toggl)', { observe: true }, () => {
+  const placeholder = $('[data-qa="channel_header__buttons"]').firstChild.parentNode;
+  const projectName = $('.p-classic_nav__team_header__team__name').textContent.trim();
```

Comment:
same as above, maybe make a curry function for these two lines (`?.textContent.trim()`) 🤷‍♂